core/modules/field_ui/field_ui.module

```diff
@@ -160,6 +158,7 @@ function field_ui_menu() {
             // rules for the bundle admin pages.
             'access callback' => '_field_ui_view_mode_menu_access',
             'access arguments' => array_merge(array($entity_type, $bundle_arg, $view_mode_position, $access['access callback']), $access['access arguments']),
+            'type' => MENU_LOCAL_TASK,
```

Comment:
This makes the form a sub-tab, but it will not actually show up as a tab because it is a dynamic menu item and there is only one tab.